src/appPackageManifest.ts

```diff
@@ -3,7 +3,7 @@ import { Transform, TransformCallback } from 'stream';
 
 import * as t from 'io-ts';
 import { failure } from 'io-ts/lib/PathReporter';
-import { pipe } from 'fp-ts/lib/pipeable';
```

Comment:
This was deprecated, and encouraged to change the usage from `pipeable` module to `function` module